docs/appendix/zowe-glossary.md

```diff
@@ -0,0 +1,188 @@
+
+* **z/OS Explorer Services**
+  
+* **Zowe Systems Services Server (ZSS)** 
+
+  The Zowe desktop delegates a number of its services to the ZSS server which it accesses through the http port 8542. ZSS is written in C and has native calls to z/OS to provide its services. For more information, see Zowe architecture | Zowe Docs
+      
+* **ZIS (Zowe Interprocess Services)**  - V2: 
+
+  it is a server but not a webserver.    
+
+* **Zowe server infrastructure**  
+
+  The glue that binds the servers together (zowe-install-packaging)       
+
+##  Installation & Configuration
+
+* **Convenience build**
+
+
+* **Instance.env** - V1 Only
+
+  The Zowe instance directory contains a file instance.env that stores the Zowe configuration data. The data is read each time Zowe is started. You can modify instance.env to configure the Zowe runtime. See https://docs.zowe.org/stable/user-guide/configure-instance-directory/#updating-the-instanceenv-configuration-file 
+
+* **Zowe runtime**
+ 
+* **Started task** 
+
+  a started task (STC) is a type of runnable/running program on z/os. Zowe started task (See also ZWESVSTC) – Does it need to go in Glossary? https://www.ibm.com/docs/en/zos/2.1.0?topic=tasks-determining-whether-use-started-task
+  Zowe has 3 actually:
+  - ZWESVSTC: v1 primary server STC
+  - ZWESLSTC: v1 HA/FT primary server STC, or the primary server STC for v2 (SVSTC goes away in v2)
+  - ZWESISTC: ZIS STC
+
+* **workspace directory**
+
+* **Zowe configuration file**
+
+  Zowe.yaml: It is a YAML file that is required to configure Zowe runtime. It can be used as an alternative to instance.env if you have to configure Zowe in more granular level. It is also required to start Zowe in high availability mode. See https://docs.zowe.org/stable/user-guide/configure-instance-directory/#creating-the-zoweyaml-file 
+
+* **Zowe instance**
+    
+* **Zowe instance directory** 
+
+  Also known as <INSTANCE_DIR>. It contains information that is specific to a launch of Zowe. It contains configuration settings that determine how an instance of the Zowe server is started, such as ports that are used or paths to dependent Java and Node.js runtimes.  The instance directory also contains log directory where different 'microservices' write trace data for diagnosis, as well as a workspace and shell scripts to start and stop Zowe.
+
+* **Zowe sample library**
+
+  (See also SZWESAMP) – Does it need to go in Glossary?
+
+* **ZWESVUSR** 
+
+  This is a started task ID used to run the PROCLIB ZWESVSTC. The task starts a USS environment using BPXBATSL that executes the core Zowe Desktop (ZLUX) node.js server, the Java API Mediation Layer, and the Z Secure Services C component. To work with USS, the user ID ZWESVUSR must have a valid OMVS segment. For more information, see System requirements | Zowe Docs
```

Comment:
~that executes the core Zowe Desktop (ZLUX) node.js server, the Java API Mediation Layer, and the Z Secure Services C component~

that executes server components such as the Application Framework, API Mediation Layer, and Z Secure Services servers.